src/accelerate/checkpointing.py

```diff
@@ -48,7 +49,7 @@ def save_accelerator_state(
     optimizers: list,
     schedulers: list,
     process_index: int,
-    scaler: GradScaler = None,
+    scaler=None,
```

Comment:
Let's leave the type hint but as a forward reference?

Reply:
Not needed since we're raising the min torch